pkg/controller/pod.go

```diff
@@ -195,13 +202,20 @@ func (pnc *PodNetworksController) handlePodUpdate(oldObj interface{}, newObj int
 
 	toAdd := exclusiveNetworks(newNetworkSelectionElements, oldNetworkSelectionElements)
 	klog.Infof("%d attachments to add to pod %s", len(toAdd), namespacedName(podNamespace, podName))
+
+	netnsPath, err := pnc.netnsPath(newPod)
+	if err != nil {
+		klog.Errorf("failed to figure out the pod's network namespace: %v", err)
+		return
+	}
```

Comment:
A `DynamicAttachmentRequest` without `PodNetNS` is legit ?

Reply:
Then why we don't return the error ?

Reply:
This is the `UpdateFunc` of the controller; how can it return errors? 
```golang
	podInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
		UpdateFunc: podNetworksController.handlePodUpdate,
	})
```

Reply:
Then maybe better send an event more than an error ?

Reply:
Could be, but events go away after an hour. I prefer to just log.